grails-app/services/org/pih/warehouse/product/ProductSupplierDataService.groovy

```diff
@@ -0,0 +1,12 @@
+package org.pih.warehouse.product
+
+import grails.gorm.services.Service
+
+@Service(value = ProductSupplier, name = "productSupplierGormService")
```

Comment:
@kchelstowski Could it be done without the name? As a:
```groovy
@Service(ProductSupplier)
interface ProductSupplierDataService {
    //...
}
```
 Is this additional name required here?

Reply:
Ok nvm, we discussed it on slack that it won't work. You can disregard this comment.